pkg/cmd/pulumi/new.go

```diff
@@ -1083,6 +1090,21 @@ func templatesToOptionArrayAndMap(templates []workspace.Template,
 	return options, nameToTemplateMap
 }
 
+// formatDesc formats the template description nicely if the term width requires it to wrap around.
+func formatDesc(prefixWidth int, maxWidth int, desc string) string {
+	descFormatted := ""
+	descWidth := maxWidth - prefixWidth
+	for idx := descWidth; idx < len(desc); idx += descWidth {
+		buffer := prefixWidth
+		if strings.LastIndex(desc[:idx], " ") != -1 {
+			idx = strings.LastIndex(desc[:idx], " ")
+			buffer += (descWidth - idx - 1)
+		}
+		descFormatted = fmt.Sprintf("%s%s%s", desc[:idx], strings.Repeat(" ", buffer), desc[idx:])
```

Comment:
This doesn't matter as much since this code isn't performance sensitive, but we could probably use a combination of `fmt.Fprintf` and `strings.Builder` here to make the concatenation cheaper:

```go
var descFormatted strings.Buildre
for ... {
  // ...
  fmt.Fprintf(&descFormatted, "%s%s%s", ...)
}
return descFormatted.String()
```